lib/commands/list-subscriptions.js

```diff
@@ -26,12 +26,16 @@ module.exports = async (req, res, next) => {
   const repositories = await getRepositories(subscriptions.filter(s => s.type === 'repo'), robot);
   const accounts = await getAccounts(subscriptions.filter(s => s.type === 'account'), robot);
 
-  if (command.text === 'list') {
-    return command.respond(new SubscriptionList(repositories, accounts));
+  if (command.text === '') {
+    // When a user types the following incomplete command /github subcribe
+
+    const response = (new Help(command.command, command.subcommand)).toJSON();
+    const list = new SubscriptionList(subscriptions, repositories, accounts);
+    response.attachments.push(list.toJSON().attachments[0]);
```

Comment:
To make this cleaner I would:

- Make a method in `SubscriptionList` that returns `list.toJSON().attachments[0]`.
- Get the result and pass that as a new constructor argument to `Help`

Reply:
This is a great suggestion! Thank you 💯